oidc_data.json

```diff
@@ -0,0 +1,157 @@
+[
```

Comment:
It'd be nice not to have to add this to the root of the repository. How is this file used? Is this a copy of [these fixtures](https://github.com/mozilla/docker-test-mozilla-django-oidc/blob/main/testprovider/fixtures.json) from the Mozilla image?

Reply:
Oh, this was intended to be the `docker/` directory. I'll move that there.

It's not a copy of the Mozilla fixtures, but additional ones — to add `localhost:8000` as a client to the test provider.

Reply:
Ah, this file was committed accidentally. The oidcprovider container is already using the `extra_fixtures.json` in that directory.

Reply:
Removed